### PR TITLE
[1LP][RFR] some test fixes related to collection v3 migration

### PR DESCRIPTION
**Pull request by izapolsk**
{{pytest: --long-running cfme/tests/services/test_provision_stack.py }}

### CODE REVIEW

**Comment by rlbabyuk** [comment]
There are some failures in PRT, could you please fix or explain them.

**Comment by izapolsk** [comment]
@rlbabyuk, zone related errors appeared because @psav brought better fix to zones and I had to remove my fix. However, I forgot to remove zone tests from PR description.
test_provisioning_stack errors look weird because several PRT runs didn't have them. I've restarted PRT.

File: cfme/tests/services/test_provision_stack.py [hunk 1]
```diff
@@ -71,7 +70,7 @@ def dialog(appliance, provider, template):
         choose_type="Text Box",
         default_text_box=service_name
     )
-    dialog = DialogCollection(appliance)
+    dialog = appliance.collections.service_dialogs
     sd = dialog.instantiate(label=dialog_name)
     tab = sd.tabs.instantiate(tab_label="Basic Information")
```
Comment (mshriver) [inline cfme/tests/services/test_provision_stack.py]: Waiting to submit review until I see some update for this tab instantiation.

Comment (izapolsk) [reply cfme/tests/services/test_provision_stack.py]: sorry, it took some time because I cannot make Selenium fill catalog right in local FF

**Pull request merged**